020-LinuxFundamentals/Student/resources/fhs.md

```diff
@@ -0,0 +1,165 @@
+
+This way /usr has its own hierarchy just like the / (root) did.
+
+#### **/usr/bin**
+
+* This is the primary directory of executable commands on the system.
+* Contains all user installed command binaries and
+* If you want to execute your scripts using a single command, you usually place them in /usr/bin/
+
+#### **/usr/sbin**
+
+* This contains user installed command binaries that can only be used by system administrators.
+
+#### **/usr/lib**
+
+* This contains the essential libraries for packages in /usr/bin and /usr/sbin just like /lib.
+
+#### **/usr/local**
+
+* This is used for all packages which are compiled manually from source by the system administrator.
+* This directory has its own hierarchy with all the bin, sbin and lib folders which contain the binaries and applications of the compiled softwares.
+
+#### **/usr/share**
+
+* Contains several architecture-independent miscellaneous files
+* Man files, word list(dictionaries) and defintion files are all included in this.
+
+**The case for /usr merge – Is there really a difference between /bin and /usr/bin?**
+
+The need for moving non-essential binaries to a different folder historically arose from a lack of space in the /bin hard disk. However, that was 1971. Today over 50 years later, we no longer face the same size problems. This has rendered two separate folders for default and user-installed binaries useless. Over time this has also caused a hodge-podge in the filesystems, with both the directory having redundant binaries which makes it confusing.
+
+For this reason, over the years, many distributions (Debian, Fedora, Ubuntu, Arch etc.) have merged /usr/bin and /bin in the same directory.
+
+Similarly /usr/sbin – /sbin and /usr/lib – /lib have been merged into the same directory to simplify the directory structure. Now the /bin folder is just a symlink to the /usr/bin directory and the same for other merges.
+
+You can read more about the discussion regarding these merges [here](https://www.freedesktop.org/wiki/Software/systemd/TheCaseForTheUsrMerge/) and [here](https://www.linux-magazine.com/Issues/2019/228/Debian-usr-Merge).
+
+### **/run**
+
+* This directory contains the metadata of the device since the last boot.
+* This includes data of all the system processes and daemons that were executed in the current session.
+* Files under this directory are cleared (removed or truncated) at the beginning of the boot process.
+
+### **/srv**
+
+* You will only ever use this directory if your device is acting as a webserver, as this directory contains all the files regarding webservers.
+* For example if host a FTP connection, all the files that need to be shared should by default go in a /srv/ftp.
```

Comment:
Change to "For example, if you host an FTP server, all the files" etc.

Reply:
Change to "For example if host an FTP connection, all the files that need to be shared should by default go in /srv/ftp."